pages/guideforms.py

```diff
@@ -55,6 +55,17 @@ def validate(self, value):
   'Write it from a web developer\'s point of view.\n\n'
   'Follow the example link below for more guidance.')
 
+# This email address pattern is more restrictive than actually allowed, but it covers most reasonable uses.
+EMAIL_ADDRESS_PATTERN = '[ ]*([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9\-\.]+)\.([a-zA-Z]{2,5}){1,25}[ ]*'
+
+EMAIL_ADDRESSES_PATTERN = '^' + EMAIL_ADDRESS_PATTERN + '(,' + EMAIL_ADDRESS_PATTERN + ')*$'
+
+MULTI_EMAIL_FIELD_ATTRS = {
+    'title':"Enter one or more comma-separated complete email addresses.",
+    'multiple': True, 
+    'placeholder': 'address1@domain.com, address2@chromium.org',
+    'pattern': EMAIL_ADDRESSES_PATTERN
```

Comment:
If we use `pattern`, should remove `multiple`?

Reply:
I believe the multiple attribute is still required because the default pattern is still used by the input element, and it needs to know that it should use the default pattern in a multi-email list wrapper pattern, which, by the way, assumes the delimiter is ','.  The pattern attribute, however, needs to do its own multi-email list wrapper.  Not sure about ignoring whitespace.

BTW, I only found this out by experimenting.  I don't see it in the MDN documentation.  The "pattern" page doesn't mention "multiple", and vice versa, the "multiple" page doesn't mention "pattern".  It does seem inconsistent, however, and perhaps this reflects a bug in Chrome.

Reply:
If we keep `multiple` then we should probably specify `type="email"` too, otherwise there is nothing to answer the question "multiple what?".

You might also test it out with and without `type` and `multiple`.   caniuse.com says that `pattern` is widely supported, and I _assume_ that `pattern` supersedes those, but it would be nice to know for sure.  If you like, we can leave that for a future PR.